<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset='utf-8'>
<title>TestNG Report</title>
<style type="text/css">
table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}
td,th {border:1px solid #009;padding:.25em .5em}
.result th {vertical-align:bottom}
.param th {padding-left:1em;padding-right:1em}
.param td {padding-left:.5em;padding-right:2em}
.stripe td,.stripe th {background-color: #E6EBF9}
.numi,.numi_attn {text-align:right}
.total td {font-weight:bold}
.passedodd td {background-color: #0A0}
.passedeven td {background-color: #3F3}
.skippedodd td {background-color: #CCC}
.skippedodd td {background-color: #DDD}
.failedodd td,.numi_attn {background-color: #F33}
.failedeven td,.stripe .numi_attn {background-color: #D00}
.stacktrace {white-space:pre;font-family:monospace}
.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}
</style>
</head>
<body>
<table cellspacing="0" cellpadding="0" class="testOverview">
<tr><th>Test</th><th>Methods<br/>Passed</th><th>Scenarios<br/>Passed</th><th># skipped</th><th># failed</th><th>Total<br/>Time</th><th>Included<br/>Groups</th><th>Excluded<br/>Groups</th></tr>
<tr><td style="text-align:left;padding-right:2em"><a href="#t1">D:\reps\Testing_Project</a></td><td class="numi">26</td><td class="numi">26</td><td class="numi">0</td><td class="numi_attn">9</td><td class="numi">0.5 seconds</td><td class="numi"></td><td class="numi"></td></tr>
</table>
<table cellspacing="0" cellpadding="0" class="methodOverview" id="summary">
<tr><th>Class</th><th>Method</th><th># of<br/>Scenarios</th><th>Start</th><th>Time<br/>(ms)</th></tr>
<tr id="t1"><th colspan="5">D:\reps\Testing_Project &#8212; failed</th></tr>
<tr class="failedodd"><td rowspan="2">Alpha_OrderingTest</td><td><a href="#m1"><b>test_rooms4</b>  </a></td><td class="numi">1</td><td>1608915547964</td><td class="numi">7</td></tr><tr class="failedodd"><td><a href="#m2"><b>test_sort_full_rooms</b>  </a></td><td class="numi">1</td><td>1608915547987</td><td class="numi">4</td></tr>
<tr class="failedeven"><td>Find_Rooms_Test</td><td><a href="#m3"><b>Test_Rooms2</b>  </a></td><td class="numi">1</td><td>1608915548010</td><td class="numi">2</td></tr>
<tr class="failedodd"><td rowspan="2">View_Rooms_Test</td><td><a href="#m4"><b>testCase1</b>  </a></td><td class="numi">1</td><td>1608915548029</td><td class="numi">1</td></tr><tr class="failedodd"><td><a href="#m5"><b>testCase5</b>  </a></td><td class="numi">1</td><td>1608915548047</td><td class="numi">1</td></tr>
<tr class="failedeven"><td>Add_Customers_Test</td><td><a href="#m6"><b>addCustomerTest3</b>  </a></td><td class="numi">1</td><td>1608915548224</td><td class="numi">1</td></tr>
<tr class="failedodd"><td rowspan="3">Delete_Customer_Test</td><td><a href="#m7"><b>testCase3</b>  </a></td><td class="numi">1</td><td>1608915548244</td><td class="numi">1</td></tr><tr class="failedodd"><td><a href="#m8"><b>testCase5</b>  </a></td><td class="numi">1</td><td>1608915548252</td><td class="numi">1</td></tr><tr class="failedodd"><td><a href="#m9"><b>testCase9</b>  </a></td><td class="numi">1</td><td>1608915548259</td><td class="numi">1</td></tr>
<tr><th colspan="5">D:\reps\Testing_Project &#8212; passed</th></tr>
<tr class="passedodd"><td>Alpha_OrderingTest</td><td><a href="#m10"><b>test_sort_full_rooms_same_name</b>  </a></td><td class="numi">1</td><td>1608915547995</td><td class="numi">3</td></tr>
<tr class="passedeven"><td rowspan="3">Find_Rooms_Test</td><td><a href="#m11"><b>Test_Rooms1</b>  </a></td><td class="numi">1</td><td>1608915548004</td><td class="numi">3</td></tr><tr class="passedeven"><td><a href="#m12"><b>Test_Rooms3</b>  </a></td><td class="numi">1</td><td>1608915548019</td><td class="numi">3</td></tr><tr class="passedeven"><td><a href="#m13"><b>Test_Rooms4</b>  </a></td><td class="numi">1</td><td>1608915548024</td><td class="numi">2</td></tr>
<tr class="passedodd"><td rowspan="3">View_Rooms_Test</td><td><a href="#m14"><b>testCase2</b>  </a></td><td class="numi">1</td><td>1608915548042</td><td class="numi">1</td></tr><tr class="passedodd"><td><a href="#m15"><b>testCase3</b>  </a></td><td class="numi">1</td><td>1608915548044</td><td class="numi">1</td></tr><tr class="passedodd"><td><a href="#m16"><b>testCase4</b>  </a></td><td class="numi">1</td><td>1608915548046</td><td class="numi">0</td></tr>
<tr class="passedeven"><td rowspan="2">Add_Customers_Test</td><td><a href="#m17"><b>addCustomerTest1</b>  </a></td><td class="numi">1</td><td>1608915548057</td><td class="numi">2</td></tr><tr class="passedeven"><td><a href="#m18"><b>addCustomerTest2</b>  </a></td><td class="numi">1</td><td>1608915548062</td><td class="numi">159</td></tr>
<tr class="passedodd"><td>Display_Empty_Rooms_Test</td><td><a href="#m19"><b>displayEmptyRoomsTest</b>  </a></td><td class="numi">1</td><td>1608915548231</td><td class="numi">1</td></tr>
<tr class="passedeven"><td>Room_Test</td><td><a href="#m20"><b>setGetNameTest</b>  </a></td><td class="numi">1</td><td>1608915548238</td><td class="numi">1</td></tr>
<tr class="passedodd"><td rowspan="6">Delete_Customer_Test</td><td><a href="#m21"><b>testCase1</b>  </a></td><td class="numi">1</td><td>1608915548242</td><td class="numi">1</td></tr><tr class="passedodd"><td><a href="#m22"><b>testCase2</b>  </a></td><td class="numi">1</td><td>1608915548243</td><td class="numi">1</td></tr><tr class="passedodd"><td><a href="#m23"><b>testCase4</b>  </a></td><td class="numi">1</td><td>1608915548249</td><td class="numi">2</td></tr><tr class="passedodd"><td><a href="#m24"><b>testCase6</b>  </a></td><td class="numi">1</td><td>1608915548255</td><td class="numi">0</td></tr><tr class="passedodd"><td><a href="#m25"><b>testCase7</b>  </a></td><td class="numi">1</td><td>1608915548256</td><td class="numi">1</td></tr><tr class="passedodd"><td><a href="#m26"><b>testCase8</b>  </a></td><td class="numi">1</td><td>1608915548257</td><td class="numi">1</td></tr>
<tr class="passedeven"><td>Integration_Testing</td><td><a href="#m27"><b>test1</b>  </a></td><td class="numi">1</td><td>1608915548278</td><td class="numi">1</td></tr>
<tr class="passedodd"><td rowspan="4">StoreDataUnitTest</td><td><a href="#m28"><b>store0Customers</b>  </a></td><td class="numi">1</td><td>1608915548291</td><td class="numi">9</td></tr><tr class="passedodd"><td><a href="#m29"><b>store1Customer</b>  </a></td><td class="numi">1</td><td>1608915548301</td><td class="numi">3</td></tr><tr class="passedodd"><td><a href="#m30"><b>store5Customers</b>  </a></td><td class="numi">1</td><td>1608915548305</td><td class="numi">4</td></tr><tr class="passedodd"><td><a href="#m31"><b>storeAllCustomers</b>  </a></td><td class="numi">1</td><td>1608915548309</td><td class="numi">3</td></tr>
<tr class="passedeven"><td rowspan="4">RetreiveDataUnitTest</td><td><a href="#m32"><b>retreive0data</b>  </a></td><td class="numi">1</td><td>1608915548321</td><td class="numi">5</td></tr><tr class="passedeven"><td><a href="#m33"><b>retreive1data</b>  </a></td><td class="numi">1</td><td>1608915548327</td><td class="numi">3</td></tr><tr class="passedeven"><td><a href="#m34"><b>retreive5data</b>  </a></td><td class="numi">1</td><td>1608915548330</td><td class="numi">4</td></tr><tr class="passedeven"><td><a href="#m35"><b>retreivealldata</b>  </a></td><td class="numi">1</td><td>1608915548335</td><td class="numi">3</td></tr>
</table>
<h1>D:\reps\Testing_Project</h1>
<h2 id="m1">View_Rooms_Test:testCase5</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: element # 9 isn&apos;t equal to e
	at org.testng.Assert.fail(Assert.java:96)
	at View_Rooms_Test.testCase5(View_Rooms_Test.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m2">Add_Customers_Test:addCustomerTest3</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: expected [Ali] but found [Ahmed]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at Add_Customers_Test.addCustomerTest3(Add_Customers_Test.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m3">Delete_Customer_Test:testCase9</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: expected [3] but found [4]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.testng.Assert.assertEquals(Assert.java:662)
	at Delete_Customer_Test.testCase9(Delete_Customer_Test.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m4">Delete_Customer_Test:testCase3</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: expected [1] but found [0]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.testng.Assert.assertEquals(Assert.java:662)
	at Delete_Customer_Test.testCase3(Delete_Customer_Test.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m5">Alpha_OrderingTest:test_rooms4</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at org.testng.Assert.assertTrue(Assert.java:54)
	at Alpha_OrderingTest.test_rooms4(Alpha_OrderingTest.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m6">View_Rooms_Test:testCase1</h2>
<div>
<div class="stacktrace">java.lang.NullPointerException
	at Menu.viewRooms(Menu.java:118)
	at View_Rooms_Test.testCase1(View_Rooms_Test.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m7">Delete_Customer_Test:testCase5</h2>
<div>
<div class="stacktrace">java.lang.NullPointerException
	at Menu.deleteCustomer(Menu.java:199)
	at Delete_Customer_Test.testCase5(Delete_Customer_Test.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m8">Find_Rooms_Test:Test_Rooms2</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at org.testng.Assert.assertTrue(Assert.java:54)
	at Find_Rooms_Test.Test_Rooms2(Find_Rooms_Test.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m9">Alpha_OrderingTest:test_sort_full_rooms</h2>
<div>
<div class="stacktrace">java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at org.testng.Assert.assertTrue(Assert.java:54)
	at Alpha_OrderingTest.test_sort_full_rooms(Alpha_OrderingTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
</div>
</div>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m10">Delete_Customer_Test:testCase4</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m11">RetreiveDataUnitTest:retreive0data</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m12">Alpha_OrderingTest:test_sort_full_rooms_same_name</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m13">RetreiveDataUnitTest:retreive5data</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m14">StoreDataUnitTest:store5Customers</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m15">Find_Rooms_Test:Test_Rooms1</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m16">Delete_Customer_Test:testCase8</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m17">Display_Empty_Rooms_Test:displayEmptyRoomsTest</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m18">StoreDataUnitTest:storeAllCustomers</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m19">View_Rooms_Test:testCase4</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m20">Delete_Customer_Test:testCase2</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m21">RetreiveDataUnitTest:retreivealldata</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m22">Add_Customers_Test:addCustomerTest2</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m23">Room_Test:setGetNameTest</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m24">StoreDataUnitTest:store0Customers</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m25">View_Rooms_Test:testCase2</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m26">View_Rooms_Test:testCase3</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m27">StoreDataUnitTest:store1Customer</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m28">Find_Rooms_Test:Test_Rooms3</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m29">Delete_Customer_Test:testCase1</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m30">Add_Customers_Test:addCustomerTest1</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m31">Delete_Customer_Test:testCase6</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m32">Delete_Customer_Test:testCase7</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m33">Integration_Testing:test1</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m34">RetreiveDataUnitTest:retreive1data</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
<h2 id="m35">Find_Rooms_Test:Test_Rooms4</h2>
<p class="totop"><a href="#summary">back to summary</a></p>
</body></html>
